bpf/lib/nodeport.h

```diff
@@ -2196,12 +2230,15 @@ nodeport_dsr_ingress_ipv4(struct __ctx_buff *ctx, struct ipv4_ct_tuple *tuple, f
 			 */
 			return DROP_INVALID;
 
+		ct_state_new.nat_addr.p4 = addr;
+		ct_state_new.nat_port = port;
 		ct_state_new.src_sec_id = WORLD_IPV4_ID;
 		ct_state_new.dsr_internal = 1;
 
 		ret = ct_create4(get_ct_map4(tuple), NULL, tuple, ctx,
 				 CT_EGRESS, &ct_state_new, ext_err);
 		if (!IS_ERR(ret))
+			/* TODO remove this in v1.20 */
```

Comment:
Should we create a GH issue for this as well as a follow up (maybe there already is)? Is there any way to mark this as a release blocker for 1.20?